modular_sand/code/game/objects/items/cosmetics.dm

```diff
@@ -39,7 +39,7 @@
 		H.facial_hair_style = new_style
 		H.update_hair()
 
-/obj/item/razor/straightrazor
+/obj/item/razor/straightrazor1
```

Comment:
This line change breaks procs and maps that previously contained this item

Reply:
I figured. I have some plans with this PR after some discussion